Cargo.toml

```diff
@@ -18,9 +18,9 @@ tokio = { version = "1.0", features = ["rt", "rt-multi-thread", "macros"] }
 metrics = { version = "0.14", optional = true }
 metrics-exporter-prometheus = { version = "0.3", optional = true }
 metrics-core = { version = "0.5", optional = true }
-metrics-runtime = { version = "0.13", optional = true }
+#metrics-runtime = { version = "0.13", optional = true }
```

Comment:
Delete the line instead here.

Reply:
Oops, I commented it to test it still compiled and just didn't remove it afterwards :|
Give me a sec